lib/internal/messages_en.js

```diff
@@ -0,0 +1,130 @@
+module.exports = {
+  "Unknown signal: {0}": {
+    "key": "ERR000001"
+  },
+  "Channel closed": {
+    "key": "ERR000002"
+  },
+  "\"message\" argument cannot be undefined": {
+    "key": "ERR000003"
+  },
+  "This handle type can't be sent": {
+    "key": "ERR000004"
+  },
+  "IPC channel is already disconnected": {
+    "key": "ERR000005"
+  },
+  "Incorrect value of stdio option: {0}": {
+    "key": "ERR000006"
+  },
+  "Child process can have only one IPC pipe": {
+    "key": "ERR000007"
+  },
+  "You cannot use IPC with synchronous forks": {
+    "key": "ERR000008"
+  },
+  "Asynchronous forks do not support Buffer input: {0}": {
+    "key": "ERR000009"
+  },
+  "Incorrect value for stdio stream: {0}": {
+    "key": "ERR000010"
+  },
+  
```

Comment:
Just visual separation for now. As I said, it's a work in progress ;-) I'm not in love with the idea of stuffing all these into a single js file